main.tf

```diff
@@ -12,7 +12,8 @@ locals {
   vpc_cidr_block    = data.terraform_remote_state.common.outputs.vpc_cidr_block
   vpc_default_sg_id = data.terraform_remote_state.common.outputs.vpc_default_sg_id
 
-  primary_domain = data.terraform_remote_state.common.outputs.primary_domain
+  primary_domain                     = data.terraform_remote_state.common.outputs.primary_domain
+  domain_openbraininstitute_org_name = data.terraform_remote_state.common.outputs.domain_openbraininstitute_org_name
```

Comment:
wait no, we don't have that variable set up

Reply:
@bilalesi you use https://github.com/openbraininstitute/aws-terraform-deployment-common/pull/32/files#diff-de6c47c2496bd028a84d55ab12d8a4f90174ebfb6544b8b5c7b07a7ee4f27ec7R140